server/src/aiController.js

```diff
@@ -0,0 +1,54 @@
+const OpenAI = require('openai');
+
+const ai = async (request, response) => {
+  const { theme, quantity, gender } = request.body;
+  const openai = new OpenAI({
+    organization: 'org-DhkAl70aE8X3otKSx9c7HKU7',
```

Comment:
This should probably be an environment variable too.

Reply:
Good point.  I'll add as an env variable.